### PR TITLE
[GIE Standalone] Add helm chart to deploy GIE on vineyard storage 

### PR DESCRIPTION
**Pull request by shirly121**
## What do these changes do?

as titled

Fixes

### CODE REVIEW

**Review by siyuan0322 — APPROVED** [review]

File: charts/gie-standalone/README.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+```
+### get service endpoint
+```
+# execute in advance if in minikube environment
+minikube tunnel 
+# gremlin endpoint
+kubectl describe svc <your-release-name>-gie-standalone-frontend | grep "Endpoints:" | awk -F' ' '{print $2}'
+```
+## Customized Config
+### download helm package
```
Comment (siyuan0322) [inline charts/gie-standalone/README.md:70]: Helm support configuring from command line or from a file.
https://helm.sh/zh/docs/helm/helm_install/
It's the recommended way to customize configurations over download the source codes.

Comment (shirly121) [reply charts/gie-standalone/README.md:70]: 嗯，我的文档没写清楚，需要从source code配置的应该是自定义的loader文件，这个我看官方helm是需要在指定目录下添加文件，最终才能被load进configMap

Comment (shirly121) [reply charts/gie-standalone/README.md:70]: > Helm support configuring from command line or from a file. https://helm.sh/zh/docs/helm/helm_install/ It's the recommended way to customize configurations over download the source codes.

嗯，我的文档没写清楚，需要从source code配置的应该是自定义的loader文件，这个我看官方helm是需要在指定目录下添加文件，最终才能被load进configMap

File: charts/gie-standalone/README.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
+# GIE Standalone Deployment
+## Prerequisite
+### etcd
+```
+# TODO: the link need to be updated after merging to main
+kubectl apply -f https://raw.githubusercontent.com/shirly121/GraphScope/add_gie_deploy/charts/gie-standalone/tools/etcd.yaml
```
Comment (sighingnow) [inline charts/gie-standalone/README.md:6]: It's ok to have `kubectl apply -f https://raw.githubusercontent.com/alibaba/GraphScope/add_gie_deploy/charts/gie-standalone/tools/etcd.yaml` before merging.

**Review by sighingnow — APPROVED** [review]
LGTM.

**Review by longbinlai — APPROVED** [review]
LGTM

**Pull request merged**